packages/cart/src/models/shipping-method.ts

```diff
@@ -24,18 +24,18 @@ export default class ShippingMethod {
   @Property({ columnType: "text" })
   cart_id: string
 
-  @ManyToOne(() => Cart, {
-    onDelete: "cascade",
+  @ManyToOne({
+    entity: () => Cart,
     index: "IDX_shipping_method_cart_id",
-    nullable: true,
+    cascade: [Cascade.REMOVE, Cascade.PERSIST],
```

Comment:
q: shouldn't we be cascading soft deletes on relations?

Reply:
Yes, I will need to make the models soft-deletable, but currently awaiting Adrien's, that will create all related methods under the hood.